Kiosk/Bid Fulfillment/LoadingViewController.swift

```diff
@@ -137,6 +137,10 @@ public class LoadingViewController: UIViewController {
         titleLabel.text = "Registration Complete"
         bidConfirmationImageView.image = UIImage(named: "BidHighestBidder")
         fulfillmentContainer()?.cancelButton.setTitle("DONE", forState: .Normal)
+        RACSignal.interval(1, onScheduler: RACScheduler.mainThreadScheduler()).take(1).subscribeCompleted { [weak self] () -> Void in
```

Comment:
The pattern of "wait X seconds"? We use that in two other places – syncing and pinging – but those are both recurring.